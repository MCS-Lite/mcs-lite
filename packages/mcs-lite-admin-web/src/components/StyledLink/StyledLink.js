import styled from 'styled-components';
import { Link } from 'react-router';

const StyledLink = styled(Link)`
  display: block;
  text-decoration: none;
  cursor: pointer;
  color: currentColor;
`;

StyledLink.displayName = 'StyledLink';

export default StyledLink;
