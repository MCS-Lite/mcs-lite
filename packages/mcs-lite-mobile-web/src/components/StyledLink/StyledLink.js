import styled from 'styled-components';
import { Link } from 'react-router';

const StyledLink = styled(Link)`
  display: block;
  text-decoration: none;
  cursor: pointer;
  color: currentColor;
`;

export default StyledLink;
