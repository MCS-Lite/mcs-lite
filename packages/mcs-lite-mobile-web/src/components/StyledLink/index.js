import StyledLink from './StyledLink';

export default StyledLink;
