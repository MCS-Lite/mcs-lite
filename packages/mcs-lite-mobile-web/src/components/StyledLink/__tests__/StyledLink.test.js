import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import StyledLink from '../';

it('should renders <StyledLink> correctly', () => {
  const wrapper = shallow(
    <StyledLink />,
  );

  expect(toJson(wrapper)).toMatchSnapshot();
});
