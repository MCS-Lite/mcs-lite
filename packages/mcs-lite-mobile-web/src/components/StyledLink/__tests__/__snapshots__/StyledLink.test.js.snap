// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders <StyledLink> correctly 1`] = `
.c0 {
  display: block;
  text-decoration: none;
  cursor: pointer;
  color: currentColor;
}

<Link
  className="c0"
  onlyActiveOnIndex={false}
  style={Object {}}
/>
`;
